src/graphql/sampling.rs

```diff
@@ -458,6 +465,7 @@ mod tests {
                         interval: FIFTEEN_MINUTES,
                         period: ONE_DAY,
                         offset: 0,
+                        node: "collector",
```

Comment:
The word "collector", as I mentioned elsewhere, doesn't quite capture the meaning we intended. We've already decided to go with a much better name: "sensor".

Reply:
Done. I replaced "collector" with "sensor" in group.rs and event.rs.